src/Type.cpp

```diff
@@ -71,7 +72,20 @@ Type *Type::void_type = NULL;
 static vector<Type *> AllTypes;
 static vector<Type *> derived_types;
 
+static vector<FunctionAttribute*> type_attributes;
+
 //////////////////////////////////////////////////////////////////////
+
+void
+GenerateAttributes()
```

Comment:
The function name is misleading, it's actually not generating any attributes.

Reply:
Yeah. It is because I have not added other attributes like `aligned, visibility....`

Reply:
It does not clarify the function name.